docs/clusters/quick-install.md

```diff
@@ -1,11 +1,13 @@
 ---
 id: quick-install
 title: How to quickly install a Temporal Cluster for testing and local development
-sidebar_label: Run a dev Cluster
+sidebar_label: Run a development Cluster
 description: There are four ways to quickly install and run a Temporal Cluster.
 ---
 
-The following sections list various methods of deploying your [Temporal Clusters](/concepts/what-is-a-temporal-cluster) locally, so that you can use and interact with the [Temporal Client](/concepts/what-is-a-temporal-client) APIs and [tctl](/tctl-v1) commands to test and develop applications.
+Temporal has various methods of working with [Temporal Clusters](/concepts/what-is-a-temporal-cluster) in a local environment. These containers are called Temporal development Clusters and are used to interact with the [Temporal Client](/concepts/what-is-a-temporal-client) APIs and [tctl](/tctl-v1) commands to test and develop applications.
```

Comment:
Not a fan of this change.
I think it was clearer before - the use of the term "containers" is not helping.